packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -20,14 +25,38 @@ export default function BottomSheetRangeCell( props ) {
 		step = 1,
 		minimumTrackTintColor = '#00669b',
 		maximumTrackTintColor = Platform.OS === 'ios' ? '#e9eff3' : '#909090',
-		thumbTintColor = Platform.OS === 'ios' ? '#fff' : '#00669b',
+		thumbTintColor = Platform.OS === 'android' && '#00669b',
 		...cellProps
 	} = props;
+	const [ accessible, onChangeAccessible ] = useState( true );
+
+	const onCellPress = () => {
+		onChangeAccessible( false );
+		if ( this.sliderRef ) {
+			const reactTag = findNodeHandle( this.sliderRef );
+			AccessibilityInfo.setAccessibilityFocus( reactTag );
+		}
+	};
```

Comment:
* What I wanted to achieve here is to make the parent component (`Cell`) not accessible, since I want to focus `Slider` and have also `TextInput` accessible as well. 

* Wdyt about having there only one component when screen reader is enabled?

* Both platforms are reading the value in percents, e.g. `100` px is is read as `17 percents`.

Reply:
@etoledom Could you please look at it? I would like to know if something is wrong there. Thanks!